### PR TITLE
Add Message ID 27

### PR DESCRIPTION
**Pull request by dcuenot**
Hello,

Please find a Pull Request, in order to handle the Message Id 27.
Spec is here: https://www.itu.int/rec/R-REC-M.1371-5-201402-I/en

KR
Damien

### CODE REVIEW

File: ais/src/main/java/au/gov/amsa/ais/message/AisPositionGPS.java [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+
+import au.gov.amsa.ais.*;
+
+import static au.gov.amsa.ais.AisMessageType.*;
+import static au.gov.amsa.ais.AisMessageType.POSITION_GPS;
+
+/**
+ * Decoder for AIS message type 27 (AIS Satelitte).
+ *
+ * @author dxm
```
Comment (davidmoten) [inline ais/src/main/java/au/gov/amsa/ais/message/AisPositionGPS.java]: nit: delete this line

File: ais/src/main/java/au/gov/amsa/ais/message/AisPositionGPS.java [hunk 2]
```diff
@@ -0,0 +1,203 @@
+package au.gov.amsa.ais.message;
+
+import au.gov.amsa.ais.*;
```
Comment (davidmoten) [inline ais/src/main/java/au/gov/amsa/ais/message/AisPositionGPS.java]: nit: unroll imports please

**Comment by dcuenot** [comment]
Minor changes pushed

**Comment by davidmoten** [comment]
Thanks @dcuenot for the contribution, greatly appreciated!

**Comment by davidmoten** [comment]
Seems some compile errors got introduced by this change. I need to turn on Travis CI for pull requests!

**Comment by dcuenot** [comment]
Sorry for the compile issues. I screwed up when I did the backport

**Comment by dcuenot** [comment]
Do you plan to do a new release with the message Id 27?
Thanks for your answer :)

**Comment by davidmoten** [comment]
I've released at 0.5.22. You'll need to build the tag yourself as we are not publishing to Maven Central at the moment.

**Pull request merged**